kafka/4.2/modules/ROOT/pages/kafka-connector-reference.adoc

```diff
@@ -331,7 +331,7 @@ Messages that appear after messages belonging to ongoing transactions are withhe
 ** MINUTES
 ** HOURS
 ** DAYS |  The time unit for the connections maximum idle time scalar. |  `SECONDS` |
-| TLS Configuration a| <<Tls>> | Defines a TLS configuration, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, you must configure at least the keystore in the `tls:context` child element of the `listener-config`. | |
+| TLS Configuration a| <<Tls>> | Defines a TLS configuration, which can be used from both the client and server sides to secure communication for the Mule app. The connector will automatically set the 'security.protocol' to use for communication. Valid values are PLAINTEXT / SSL / SASL_PLAINTEXT / SASL_SSL. Default value when no configuration has been provided is PLAINTEXT(or SASL_PLAINTEXT for SASL authentication - kerberos/scram). If SSL was configured as protocol on the broker side then the user needs to configure at least the keystore in the 'tls:context' child element of this config and the connector will automatically use SSL(or SASL_SSL for SASL authentication) as 'security.protocol'. | |
```

Comment:
Here shows _kerberos/scram_ and in the previous description shows _kerberos/scram/plain_ is this intended?

Reply:
Updated thanks for pointing it out.